stream_alert_cli/helpers.py

```diff
@@ -59,6 +59,9 @@ def run_command(runner_args, **kwargs):
     except subprocess.CalledProcessError as err:
```

Comment:
@jacknagz `OSError` does not have a `cmd` property - so we could, but we would lose some granularity. Doing it this way has the added bonus of knowing what command caused the `OSError` (see: `runner_args[0]`)